src/Data/Nat/Properties.agda

```diff
@@ -1116,6 +1111,59 @@ n⊔m≡m⇒n≤m n⊔m≡m = subst (_ ≤_) n⊔m≡m (m≤m⊔n _ _)
 n⊔m≡n⇒m≤n : ∀ {m n} → n ⊔ m ≡ n → m ≤ n
 n⊔m≡n⇒m≤n n⊔m≡n = subst (_ ≤_) n⊔m≡n (n≤m⊔n _ _)
 
+m⊓n≤m⊔n : ∀ m n → m ⊔ n ≤ m ⊔ n
```

Comment:
The name does not match the type. Presumably you want `m ⊓ n ≤ m ⊔ n` because this is just
(extensionally) `≤-refl (m ⊔ n)`

Reply:
That bug has been in deployment for a while *eek*